apm-collector/apm-collector-worker/src/main/java/org/skywalking/apm/collector/worker/httpserver/AbstractPost.java

```diff
@@ -2,60 +2,46 @@
 
 import com.google.gson.JsonObject;
 import com.google.gson.stream.JsonReader;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.skywalking.apm.collector.actor.*;
-import org.skywalking.apm.collector.worker.segment.entity.Segment;
-
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
 import java.io.BufferedReader;
-import java.io.IOException;
+import org.skywalking.apm.collector.actor.AbstractLocalSyncWorker;
+import org.skywalking.apm.collector.actor.ClusterWorkerContext;
+import org.skywalking.apm.collector.actor.LocalSyncWorkerRef;
+import org.skywalking.apm.collector.actor.LocalWorkerContext;
+import org.skywalking.apm.collector.actor.Role;
+import org.skywalking.apm.collector.actor.WorkerRef;
+import org.skywalking.apm.collector.worker.instance.entity.RegistryInfo;
+import org.skywalking.apm.collector.worker.instance.entity.HeartBeat;
+import org.skywalking.apm.collector.worker.segment.entity.Segment;
 
 /**
  * @author pengys5
  */
 
-public abstract class AbstractPost extends AbstractLocalAsyncWorker {
+public abstract class AbstractPost extends AbstractLocalSyncWorker {
```

Comment:
@pengys5 , I think this class has been refactor in your new version collector. Please send your pr first.